### ISSUE TITLE
LR and False Positives Inquiry

### ISSUE THREAD

**Issue by andeees** [RangiLyu/nanodet#276]
Hello, I have two brief questions:

- When training w/ this [**cfg**](https://github.com/RangiLyu/nanodet/blob/main/config/nanodet-m.yml) I get lots of False Positives. Is there any recipe (besides from more data) to mitigate this? (i.e. changing loss weights?)
- I was surprised to see that high **learning rate** was used for many epochs before reducing it with a scheduler. Was this found empirically to work better? (i.e. start with lr=0.14)

**Comment by RangiLyu** [comment]
1. This is a common problem when using Focal Loss and quality aware method(like GFL in NanoDet). One way to solve this is to use some target sampling method, but it is hightly relevant to your data. So I can not give a specific solution.
2. One reason to use such huge lr is because the batch size is very large here(192 per gpu). And I fond that using high learning rate to train a very long time before lr decay can improve the final mAP.

BTW, the hyperparameters in the config may not be the best because I do not have resources to search them. If anyone find a better setting on COCO, welcome PR.

**Comment by andeees** [comment]
@RangiLyu thanks for the reply! Final doubt:

2. I am a little confused, are gradients summed instead averaged across batches then? (when using sum generally lr must be adjusted based on batch size).

I will try different hyper-parameters for my custom dataset and try to alleviate (1). Thanks :)
